test/utils/link_creator.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+rm -f uri_dump;
```

Comment:
Hmm, the whole script assumes the user's CWD is the restbase repo's root. No bueno. Better for the script to `cd` into it before doing anything: `cd ${DIR}/../..`